File: VB.WinForms.NET6/VB.WinFormsStarter/Form1.vb
Imports AnyCAD.Foundation

Public Class Form1

    Private mRenderView As AnyCAD.Forms.RenderControl

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        GlobalInstance.Initialize()
        mRenderView = New AnyCAD.Forms.RenderControl(SplitContainer1.Panel2)
    End Sub

    Private Sub Form1_FormClosed(sender As Object, e As FormClosedEventArgs) Handles MyBase.FormClosed
        GlobalInstance.Destroy()
    End Sub
End Class

File: WinForms.NET6/WinFormsStarter/Form1.cs
using AnyCAD.Forms;
using AnyCAD.Foundation;

namespace WinFormsStarter
{
    public partial class Form1 : Form
    {
        RenderControl mRenderView; 
        public Form1()
        {
            InitializeComponent();
            mRenderView = new RenderControl(this.splitContainer1.Panel2);
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            var shape = ShapeBuilder.MakeSphere(new GPnt(0, 0, 0), 10);
            mRenderView.ShowShape(shape, ColorTable.PaleVioletRed);
        }
    }
}